install_hooks.py

```diff
@@ -16,25 +59,60 @@ def get_init_scripts(config):
     elif 'bsd' in sys.platform or 'dragonfly' in sys.platform:
         data_files = data_files + "\nusr/local/etc/rc.d = bin/rc.d/*"
     else:
-        raise "Unsupported platform, sorry"
-        data_files = []
+        raise Exception("Unsupported platform, sorry")
+
     config['files']['data_files'] = data_files
 
+    for line in config['files']['data_files'].split('\n'):
+        line = line.strip().split('=')
+        print("Installable directories/files: %s" % line)
+
+    # Check Alignak recommended user existence
+    if not user_exists('alignak'):
+        print("The user account 'alignak' does not exist on your system. " \
+              "You must create this user on your system to proceed with " \
+              "Alignak installation.")
+        raise Exception("Installation stopped")
+
+    if not group_exists('alignak'):
+        print("The user group 'alignak' does not exist on your system. " \
+              "You must create this users group on your system to proceed with "
+              "Alignak installation.")
+        raise Exception("Installation stopped")
+
 
 def fix_alignak_cfg(config):
-    """ Fix paths, user and group in alignak.cfg and daemons/*.ini """
+    """
+    Fix paths, user and group in alignak.cfg and daemons/*.ini
+    Called one all files are copied.
+
+    :param config:
+    :return:
+    """
     default_paths = {
+        'workdir': '/var/run/alignak',
+        'logdir': '/var/log/alignak',
+        # TODO: confirm is is unuseful...
+        'modules_dir': '/var/lib/alignak/modules',
+        'plugins_dir': '/var/libexec/alignak',
+
         'lock_file': '/var/run/alignak/arbiterd.pid',
         'local_log': '/var/log/alignak/arbiterd.log',
         'pidfile': '/var/run/alignak/arbiterd.pid',
-        'workdir': '/var/run/alignak',
-        'pack_distribution_file': '/var/lib/alignak/pack_distribution.dat',
-        'modules_dir': '/var/lib/alignak/modules',
+
+        'pack_distribution_file': '/var/lib/alignak/pack_distribution.dat'
+    }
+
+    default_macros = {
+        'LOGSDIR': '/var/log/alignak',
+        'PLUGINSDIR': '/var/libexec/alignak',
```

Comment:
Any documentation on the /var/libexec path? I am not sure this is a existing path (vs /var/lib/**)